layouts/donate/section.html

```diff
@@ -33,13 +33,13 @@ <h2 class="f4 fw6 tl">Help us by funding the next hub!</h2>
           </div>
           <div class="w1">
           </div>
-          <div id="fifty-monthly" onclick="setAmount('fifty-monthly')" class="button  pa0 br0 ba b--moon-gray ph3 pv3 pointer hover-b--blue">
-            <span class="ma0 fw4 sans-serif">$50</span>
+          <div id="thirty-monthly" onclick="setAmount('thirty-monthly')" class="button  pa0 br0 ba b--mid-gray ph3 pv3 pointer hover-b--blue">
```

Comment:
Can you change this to `class="button pa0 br0 ba b--blue bg-blue-50 ph3 pv3 pointer"`? This will give the button the correct selected appearance.